### PR TITLE
Feat - actix web validator support

### PR DESCRIPTION
**Pull request by rlebran**
Hi,

I would like to bring [validator](https://docs.rs/validator/latest/validator/) support to paperclip.
To do so I use [actix-web-validator](https://github.com/rambler-digital-solutions/actix-web-validator) as a dependency.

I did add this feature for both actix3 and actix4. I personally use it for month now in the company I'm working for.

### CODE REVIEW

**Comment by tiagolobocastro** [comment]
Sorry I missed this till now, I'll try to have a look tomorrow/weekend.

**Comment by rlebran** [comment]
Hey ! 

Just for your information, I made this a while ago and never took time to get back to it.
After a tiny bit of thinking I found a way to simplify it, reusing the `Json<T>` implementation of `OperationModifier` for `ValidatedJson` and `Path` implementation for `ValidatedPath`. 

This should be easier to maintain in the future.

File: core/src/v2/actix.rs [hunk 1]
```diff
@@ -337,6 +347,45 @@ where
     }
 }
 
+#[cfg(all(
+    any(feature = "actix4-validator", feature = "actix3-validator"),
+    feature = "nightly"
```
[the remark this answers is not in the document]
Comment (rlebran) [reply core/src/v2/actix.rs:352]: To be honest I didn't know why I need this but I wanted to have the exact same logic as `Json<T>`.
If you look [here](https://github.com/paperclip-rs/paperclip/blob/9854952920d0c5b2be05db7211ceba6b54bf2f40/core/src/v2/actix.rs#L292) you will see the exact same logic. 
If you want me to remove it I can do it.

[the remark this answers is not in the document]
Comment (tiagolobocastro) [reply core/src/v2/actix.rs:352]: hmm, I think I see now, if T does not implement Apiv2Schema we add an empty schema?
I'd say leave it there then.

[the remark this answers is not in the document]
Comment (rlebran) [reply core/src/v2/actix.rs:352]: Might be but weirdly only work for nightly. I will look into it and if I find something I might do another PR.

**Comment by tiagolobocastro** [comment]
thanks @rlebran :+1:

**Pull request merged**